docs/userguide/main.md

```diff
@@ -1167,4 +1167,47 @@ spec:
         serverName: "testserver.test.local"
         insecureSkipVerify: true
   # ...
+
+### Managing ISM policies with Kubernetes resources
+
+The operator provides a custom Kubernetes resource that allow you to create/update/manage ISM policies using Kubernetes objects.
+
+#### OpenSearch ISM policy
+
+It is possible to manage OpenSearch ISM policies in Kubernetes with the operator. The operator will not modify policies that already exist. You can create an example policy as follows:
```

Comment:
Maybe add a sentence about how the fields of the CRD correspond to the ISM policy structure in opensearch?

Reply:
^

Reply:
I have added one line, as there is mostly a direct mapping, not sure how one-liner can communicate that. Please check .